functionary/ui/templates/base.html

```diff
@@ -45,124 +41,93 @@
+                                    <i class="fa fa-lg fa-cubes fa-fw"></i>
+                                    <span class="ms-1 d-none d-md-inline">Packages</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:function-list' %}">
+                                    <i class="fa fa-lg fa-cube fa-fw"></i>
+                                    <span class="ms-1 d-none d-md-inline">Functions</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:workflow-list' %}">
+                                    <i class="fa fa-lg fa-diagram-next fa-fw"></i>
+                                    <span class="ms-1 d-none d-md-inline">Workflows</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:scheduledtask-list' %}">
+                                    <i class="fa fa-lg fa-clock fa-fw"></i>
+                                    <span class="ms-1 d-none d-md-inline">Schedules</span>
+                                </a>
+                                {% if user.is_superuser %}
+                                    <a class="nav-link px-0" href="{% url 'ui:environment-list' %}">
+                                        <i class="fa fa-lg fa-building fa-fw"></i>
+                                        <span class="ms-1 d-none d-md-inline">Environments</span>
+                                    </a>
+                                    <a class="nav-link px-0" href="{% url 'ui:team-list' %}">
+                                        <i class="fa fa-lg fa-users fa-fw"></i>
+                                        <span class="ms-1 d-none d-md-inline">Teams</span>
+                                    </a>
+                                {% endif %}
+                            </nav>
+                            <hr/>
+                            <nav class="nav nav-pills flex-column mb-5 align-items-center align-items-md-start">
+                                <a class="nav-link px-0" href="{% url 'socialaccount_connections' %}">
+                                    <i class="fa fa-lg fa-user fa-fw"></i>
+                                    <span class="ms-1 d-none d-md-inline">{{ user }}</span>
+                                </a>
                                 <form method="post" action="{% url 'account_logout' %}">
                                     {% csrf_token %}
-                                    <button class="button is-underlined is-ghost has-text-white pr-0 pb-0"
-                                            type="submit">
-                                        <span class="icon has-text-danger mr-2"><i class="fa fa-door-open"></i></span>Logout
+                                    <button class="btn nav-link px-0" type="submit">
+                                        <i class="fa fa-md fa-door-open fa-fw text-primary"></i>
```

Comment:
This icon is smaller than the others, which makes it not line up nicely with all of the other menu items.